app/controllers/errors_controller.rb

```diff
@@ -1,4 +1,6 @@
 class ErrorsController < ApplicationController
+  alias_method :current_user, :current_member
```

Comment:
nu in nici un caz nu are sens sa faci asa ceva ... asta ai adaugat-o peste tot in toate controllerele care e acelasi lucru cu adaugarea ei in application_controller. Daca te uiti errors_controller depinde de application_controller deci de ce naibai iti trebuie. A doua situatie e ca nu trebuie sa faci load_resource in controllerele astea pentru ca nu iti trebuie nici un @issue sau @website etc. poti sa faci authorize_resource daca vrei dar nu load nu are sens. Eu in locul tau nu as baga deloc asa ceva as sterge tooate astea. In plus ti-am zis ca ne trebuie doar in api ...